src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala

```diff
@@ -276,6 +282,18 @@ case class AggregatedBreakpointPileup(id: String,
     assert(pileup.right_contig == right_contig)
     assert(pileup.left_strand == left_strand)
     assert(pileup.right_strand == right_strand)
+
+    val left_targets = (this.left_targets, pileup.left_targets) match {
+      case (_, None) => this.left_targets
+      case (None, _) => pileup.left_targets
+      case (Some(this_left), Some(pileup_left)) => Some(f"${this_left},${pileup_left}")
+    }
+    val right_targets = (this.right_targets, pileup.right_targets) match {
+      case (_, None) => this.right_targets
+      case (None, _) => pileup.right_targets
+      case (Some(this_right), Some(pileup_right)) => Some(f"${this_right},${pileup_right}")
+    }
```

Comment:
This seems like it will lead to a lot of repetition if you end up aggregating multiple pileups that overlap the same target(s).  It seems like ideally you'd want to keep a list/set of names, and then take the union during add().  That, or tokenize them both each time, take the unique set, and then re-CSV them.

Reply:
Perhaps it's time we added `Iterable` round-tripping in Metric?